instructor/common.py

```diff
@@ -415,81 +427,113 @@ def assignments_edit_treatments(request):
             'input_headers': input_headers,
             'has_temperature': assignment.has_temperature,
             'has_collection': assignment.has_collection_time,
-            'assignment': assignment
+            'assignment': assignment,
+            'assignment_name': assignment.name
```

Comment:
Same comment here regarding using `assignment.name` in place of `assignment` in template